docs/en/enterprise-edition/content-collections/administration/configure-external-integrations-on-prisma-cloud/integrate-prisma-cloud-with-servicenow.adoc

```diff
@@ -16,6 +20,7 @@ If you are using a ServiceNow developer instance, make sure that it is not hiber
 . xref:integrate-prisma-cloud-with-servicenow.adoc#idc4548ecb-5da3-4de2-8072-7f0c3df02de3[Enable the ServiceNow Integration on Prisma Cloud]
```

Comment:
I think we need a lead-in sentence here, something like, "Follow the steps below to integrate PC with SNOW."

Reply:
We already have a leadin sentence under Enable the ServiceNow Integration on Prisma Cloud.